packages/core/src/classes.ts

```diff
@@ -267,3 +267,21 @@ export interface Version extends Doc {
   minor: number
   patch: number
 }
+
+/**
+ * @public
+ */
+export interface Backlink extends AttachedDoc {
+  // A target document
+  // attachedTo <- target document we point to
+  // A target document class
+  // attachedToClass
+
+  message: string
+  // Source document we have reference from, it should be parent document for Comment/Message.
+  backlinkId: Ref<Doc>
+  // Source document class
+  backlinkClass: Ref<Class<Doc>>
+  // Reference to comment documentId
+  attachedDocId?: Ref<Doc>
```

Comment:
Why we need this one?

Reply:
It is a comment id, for example, if we are adding comment to a card, `backlinkId` would be card id, `attachedDocId` would be a comment id

Reply:
1. We could add Backlink to card if card has field marked with MarkupWithRef, like description and only backlinkId, backlinkClass is required.
2. If Card has comment, comment itself will be backlinkId and backlinkClass.
3. Same for any checkitemlist/checkitem attached to card.
So I do not see this field is required.